rstudio/c9s-python-3.9/Dockerfile

```
@@ -45,6 +45,11 @@ RUN chmod 1777 /var/run/rstudio-server && \
     mkdir -p /usr/share/doc/R
 COPY rsession.conf /etc/rstudio/rsession.conf
 
+# package installation 
+RUN dnf install -y libsodium-devel.x86_64 libgit2-devel.x86_64 libcurl-devel cmake "flexiblas-*"
```

```suggestion
RUN dnf install -y libsodium-devel.x86_64 libgit2-devel.x86_64 libcurl-devel harfbuzz-devel.x86_64 fribidi-devel.x86_64 cmake "flexiblas-*"
```

In addition to the R packages 'tidyverse,' 'tidymodels,' 'plumber,' 'vetiver,' and 'devtools,' it is essential to include the following OS packages for a successful installation: 'harfbuzz-devel.x86_64' and 'fribidi-devel.x86_64.'